src/actions/mover.ts

```diff
@@ -140,6 +146,47 @@ export class Mover {
     // (otherwise dragging will break).
     getFocusManager().focusNode(block);
     block.getFocusableElement().addEventListener('blur', blurListener);
+
+    // Register a keyboard shortcut under the key combos of all existing
+    // keyboard shortcuts that commits the move before allowing the real
+    // shortcut to proceed. This avoids all kinds of fun brokenness when
+    // deleting/copying/otherwise acting on a block in move mode.
+    const shortcutKeys = Object.values(ShortcutRegistry.registry.getRegistry())
+      .flatMap((shortcut) => shortcut.keyCodes)
+      .filter((keyCode) => {
+        return (
+          keyCode &&
+          ![
+            utils.KeyCodes.RIGHT,
+            utils.KeyCodes.LEFT,
+            utils.KeyCodes.UP,
+            utils.KeyCodes.DOWN,
+            utils.KeyCodes.ENTER,
```

Comment:
Also needs `KeyCodes.SPACE` from the finish move action alongside `ENTER`.

Potentially these could be pulled from a list of the move actions from `move.ts`? Would be helpful in future as we hope to add UI the ability to rebind shortcuts (though these ones are perhaps less likely to change than some).

Reply:
And we do anticipate that when all of the keyboard nav code makes it into core, the shortcut actions will directly cancel moves. This setup is necessary because some shortcuts are in core and some are in the plugin. We will keep UI rebinding in mind for design work on that.

Reply:
We don't actually need space, because this shortcut finishes the move itself; Enter is there to support command-enter to bring up the contextual menu, not to commit the move. But we do need Escape to abort the move, so I added that in (and cleaned up this shortcut in the new common cleanup method introduced in #561).